hatchery/paymodels.go

```diff
@@ -13,14 +14,32 @@ import (
 
 var ErrNopaymodels = errors.New("no paymodels found")
 
+var getPayModelTableCreds = func(sess *session.Session) aws.Config {
+	// credentials for pay model dynamodb table
+	var awsConfig aws.Config
+
+	// Assume a new role if we have a pay model arn
+	if Config.Config.PayModelsDynamodbArn != "" {
+		fmt.Println("Got the ARN for assume role")
```

Comment:
lets remove this print since it will be called every time status check happens, and it is not really help message